conntrack_linux.go

```diff
@@ -159,13 +159,18 @@ func (h *Handle) ConntrackDeleteFilter(table ConntrackTableType, family InetFami
 // ConntrackDeleteFilters deletes entries on the specified table matching any of the specified filters using the netlink handle passed
 // conntrack -D [table] parameters         Delete conntrack or expectation
 func (h *Handle) ConntrackDeleteFilters(table ConntrackTableType, family InetFamily, filters ...CustomConntrackFilter) (uint, error) {
+	var errMsgs []string
 	res, err := h.dumpConntrackTable(table, family)
 	if err != nil {
-		return 0, err
+		if  !errors.Is(err, ErrDumpInterrupted)  {
```

Comment:
There's a small formatting issue here:

```suggestion
		if !errors.Is(err, ErrDumpInterrupted) {
```

(just to make sure no one will have to commit code-style fixes)

Reply:
doh!  That was me probably; my spacebar is a bit borked and has a tendency of double-firing.

if you can squash the commits again, @adrianmoisey that'd be great (sorry for that!)

Reply:
Haha, no problem. It's also me being lazy and relying on my IDE a little too much 😉